pkg/document/document_test.go

```diff
@@ -19,8 +19,11 @@ package document_test
 import (
 	"errors"
 	"fmt"
+	"runtime"
 	"testing"
 
+	"github.com/yorkie-team/yorkie/pkg/document/json"
+
 	"github.com/stretchr/testify/assert"
 
 	"github.com/yorkie-team/yorkie/api/converter"
```

Comment:
Please group packages in the following order.

- Standard library
- 3rd party library
- Internal packages

https://github.com/uber-go/guide/blob/master/style.md#import-group-ordering

Reply:
I sent a PR related to Import Group Ordering.
https://github.com/yorkie-team/yorkie/pull/117/files

Reply:
Thanks for the feedback.
I seem to make this mistake often. I think I can solve this problem thanks to the PR you sent us. :)

Reply:
This proposal has been modified from b52d78d. Please check.